cobalt/site/docs/development/setup-linux.md

```diff
@@ -19,10 +19,10 @@ Required libraries can differ depending on your Linux distribution and version.
 
     ```
     $ sudo apt update && sudo apt install -qqy --no-install-recommends \
-        pkgconf ninja-build bison nasm yasm binutils clang libgles2-mesa-dev \
-        mesa-common-dev libpulse-dev libavresample-dev libasound2-dev \
-        libxrender-dev libxcomposite-dev libxml2-dev curl git \
-        python3.8-venv libxi-dev
+        pkgconf ninja-build bison nasm binutils clang libgles2-mesa-dev \
+        mesa-common-dev libpulse-dev libasound2-dev libxrender-dev \
+        libxcomposite-dev libxml2-dev curl git python3.11-venv libxi-dev \
```

Comment:
The page above mentions "These instructions were tested on a fresh ubuntu:20.04 Docker image" - can we validate this runs ? Should be as easy as checking with `docker run ubuntu:20.04`

Reply:
Specifically, python3.x should match up.  We need higher than 3.7 but whatever comes from that particular image should match.

Reply:
I kept the original version here instead. We do support 3.8 and on ubuntu 20.04 only 3.9 is available. With the disclaimer above the instructions I believe we are good.

Reply:
I should add that I tested the entire flow on a clean ubuntu 20.04 and got it all to work.